helix-term/src/commands.rs

```diff
@@ -4457,6 +4458,12 @@ fn insert_register(cx: &mut Context) {
     })
 }
 
+fn clear_registers(cx: &mut Context) {
+    cx.editor.autoinfo = None;
```

Comment:
This line can be removed: it's used in the other `*_register` commands because they add autoinfo and then clear it in the on-next-key callback

Reply:
Done.